### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by SerhiiNalyvaiko**
- [DEMO LINK](https://SerhiiNalyvaiko.github.io/layout_catalog/)
- [TEST REPORT LINK](https://SerhiiNalyvaiko.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

**Comment by SerhiiNalyvaiko** [comment]
- [DEMO LINK](https://SerhiiNalyvaiko.github.io/layout_catalog/)
- [TEST REPORT LINK](https://SerhiiNalyvaiko.github.io/layout_catalog/report/html_report/)

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,313 @@
...
+            </div>
+            <div class="card__price">
+              <p class="card__price-text">Price:</p>
+              <p class="card__price-total">$2,199</p>
+            </div>
+            <a
+              href="#"
+              class="card__button-link"
+              data-qa="card-hover"
+            >buy</a>
```
Comment (tetiana-ivanchenko) [inline src/index.html:82]:
```suggestion
            <a
              href="#"
              class="card__button-link"
              data-qa="card-hover"
            >
              buy
            </a>
```

Comment (tetiana-ivanchenko) [reply src/index.html:82]: fix it everywhere

File: src/styles/card.scss [hunk 2]
```diff
@@ -0,0 +1,99 @@
+.card {
+  flex-wrap: wrap;
```
Comment (tetiana-ivanchenko) [inline src/styles/card.scss:2]: redundant

File: src/styles/card.scss [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 0;
+  border: 1px solid #f3f3f3;
+  color: #060b35;
+  background-color: #fff;
```
Comment (tetiana-ivanchenko) [inline src/styles/card.scss:9]: redundant

File: src/styles/card.scss [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+  &__title {
+    display: flex;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__product-text {
+    font-weight: 400;
```
Comment (tetiana-ivanchenko) [inline src/styles/card.scss:36]: it's default value

File: src/styles/card.scss [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    margin-bottom: 16px;
+  }
+
+  &__rewiews {
+    margin-bottom: 24px;
+    display: flex;
+    flex-direction: row;
```
Comment (tetiana-ivanchenko) [inline src/styles/card.scss:46]: it's default value

File: src/styles/card.scss [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  &__rewiews-text {
+    display: flex;
+    flex-direction: row;
```
Comment (tetiana-ivanchenko) [inline src/styles/card.scss:53]: it's default value

File: src/styles/header-nav.scss [hunk 7]
```diff
@@ -0,0 +1,46 @@
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
```
Comment (tetiana-ivanchenko) [inline src/styles/header-nav.scss:5]: it's default value

File: src/styles/header-nav.scss [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+    align-items: center;
+    text-decoration: none;
+    height: 60px;
+    margin-left: 20px;
+    text-transform: uppercase;
+    color: #060b35;
+  }
+
+  &__link:hover {
+    color: #00acdc;
```
Comment (tetiana-ivanchenko) [inline src/styles/header-nav.scss:26]: move repeatable colors in the variables

File: src/styles/header.scss [hunk 9]
```diff
@@ -0,0 +1,14 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
```
Comment (tetiana-ivanchenko) [inline src/styles/header.scss:5]: it's default value

File: src/styles/html.scss [hunk 10]
```diff
@@ -0,0 +1,7 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  background-color: #fff;
```
Comment (tetiana-ivanchenko) [inline src/styles/html.scss:4]: it's default value

File: src/styles/html.scss [hunk 11]
```diff
@@ -0,0 +1,7 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
```
Comment (tetiana-ivanchenko) [inline src/styles/html.scss:3]: it's default value

File: src/styles/reset-css.scss [hunk 12]
```diff
@@ -0,0 +1,16 @@
...
+  box-sizing: border-box;
+}
+
+body,
+div,
+h1,
+h2,
+p {
+  margin: 0;
+  background-color: #fff;
```
Comment (tetiana-ivanchenko) [inline src/styles/reset-css.scss:11]: why did you set white background to almost all elements? it's white by default

File: src/styles/reset-css.scss [hunk 13]
```diff
@@ -0,0 +1,16 @@
...
+}
+
+a {
+  text-decoration: none;
```
Comment (tetiana-ivanchenko) [inline src/styles/reset-css.scss:15]: reset it at the class selector for this element

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]
fix tests, your image should be centered horizontally

File: src/styles/header.scss [hunk 14]
```diff
@@ -0,0 +1,12 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  max-width: 1200px;
+
+  &__logo {
+    height: 40px;
+    width: 40px;
```
Comment (Anastasiia-Svintsova) [inline src/styles/header.scss:10]: Would be great to create a variable for logo size

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
- 8 failed tests are a lot. Please, fix them. If you have any questions ask them in the fe_chat
image

**Review by nazarmatsevych — APPROVED** [review]

**Pull request closed without merging**